microcosm_flask/paging.py

```diff
@@ -96,10 +99,25 @@ def to_tuples(self):
             ("offset", self.offset),
             ("limit", self.limit),
         ] + [
-            (key, str(self.rest[key]))
+            (key, self.to_value(self.rest[key]))
             for key in sorted(self.rest.keys())
         ]
 
+    def to_value(self, value):
+        """
+        Cast query string parameter value to string for commonly used
+        types such as enumeration values and UUIDs.
+
+        :param value - a query parameter value
+        :returns value cast to string if value is one of pre-specified types (UUID, Enum)
+            or the value as-is otherwise.
+
+        """
+        if isinstance(value, (UUID, Enum)):
+            return str(value)
+
+        return value
```

Comment:
ping @jessemyers I ended up adding this `to_value` method as a unit-test was failing here when trying to pass enum and UUID objects into the `from_query_string` factory and expect to see them serialized properly into a JSON response (as strings). 

1. I'm not sure this is a case we care about so perhaps unit-test is off?

2. In any case, I think Enum/UUID covers most mainstream use-cases for non offset/limit parameters so this would preserve backward-compatibility pretty well. removing this might break a bunch of services in hard to debug ways.. WDYT?